Testing/ORKTest/ORKTest/ORKESerialization.m

```diff
@@ -566,12 +566,13 @@ @implementation ORKESerializer
           })),
   ENTRY(ORKTextChoice,
         ^id(NSDictionary *dict, ORKESerializationPropertyGetter getter) {
-            return [[ORKTextChoice alloc] initWithText:GETPROP(dict, text) detailText:GETPROP(dict, detailText) value:GETPROP(dict, value)];
+            return [[ORKTextChoice alloc] initWithText:GETPROP(dict, text) detailText:GETPROP(dict, detailText) value:GETPROP(dict, value) exclusive:[GETPROP(dict, exclusive) boolValue]];
```

Comment:
Yes, the `ORKJSONSerializationTests` tests all pass.  The `ResearchKitTests` tests all pass as well.